assets/src/edit-story/components/panels/design/textStyle/font.js

```diff
@@ -101,6 +102,13 @@ function FontControls({ selectedElements, pushUpdate }) {
     [fontStyle, handleSelectFontWeight, maybeEnqueueFontStyle, selectedElements]
   );
 
+  const previousFontSize = usePrevious(fontSize);
+  useEffect(() => {
+    if (fontSize < 0) {
+      pushUpdate({ fontSize: previousFontSize }, true);
+    }
+  }, [fontSize, previousFontSize, pushUpdate]);
+
```

Comment:
This is dope! Should we just make this into a blur event handler? I think this gives us an extra submit when we change the font size which may cause a re-render? Tell me if I'm crazy though.

I think that using a blur event handler instead will make us run less updates.

Reply:
existing setup is working for me fine, but also happy with route @samwhale is proposing. If I did go down that route, I'd just incorporate this logic into the presubmit handler which I think should take care of it 🤷

Reply:
I do think this is cleaner:
```
const previousFontSize = usePrevious(fontSize);
  usePresubmitHandler(
    ({ fontSize: newFontSize }) => ({
      fontSize:
        newFontSize < 0
          ? previousFontSize
          : clamp(newFontSize, MIN_MAX.FONT_SIZE),
    }),
    [fontSize]
  );
```
Had to include `fontSize` in the deps there and still need to store the previous font size as far as I could tell, so I think we're still updating but I think this is still better. Works too. The only reason I'm sorta against `onBlur` is because this is such similar functionality to the `clamp`ing we're already doing here so that concern is shared by this existing functionality.

Reply:
Here's a vid of it still working, but also an example of the weird ux. What do you guys think?
https://user-images.githubusercontent.com/41136059/117515582-8a24af00-af4b-11eb-99b6-9e5dc2814676.mp4

Reply:
Also using presubmit makes the element's frame act weird. Don't worry about approving this one this week. :) I wanna make sure the ux is right here, seems kinda janky to me right now.